File: ClientApp/src/app/interfaces/product/response/warrantyPreviewResponse.ts
import { BaseResponse } from "../../base-response";

export interface WarrantyPreviewResponse extends BaseResponse {
	id: string;
	name: string | null;
	type: string;
	scopeType: string;
	status: string | null;
	period: string;
	description: string;
	conditions: string;
}

File: ClientApp/src/app/interfaces/Product/Response/WarrantyPreviewResponse.cs
namespace Contracts.Model.Product.Response;

public class WarrantyPreviewResponse
{
	public Guid Id { get; set; }
	public string Name { get; set; }
	public string Type { get; set; }
	public string Status { get; set; }
	public int Period { get; set; }
	public string Description { get; set; }
	public string Conditions { get; set; }
}